src/Sulu/Bundle/RouteBundle/Routing/RouteProvider.php

```diff
@@ -96,7 +96,7 @@ public function __construct(
     public function getRouteCollectionForRequest(Request $request)
     {
         $collection = new RouteCollection();
-        $path = $request->getPathInfo();
+        $path = rawurldecode($request->getPathInfo());
```

Comment:
should we link to: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Routing/Matcher/UrlMatcher.php#L91 
here to document why this is needed /cc @danrot